### PR TITLE
crawl and scrape color of change site

### PR DESCRIPTION
**Pull request by areeta**
firebase has been updated with all of the petitions from color of change

### CODE REVIEW

File: src/api/app.py [hunk 1]
```diff
@@ -21,10 +23,48 @@
 app = Flask(__name__)
 CORS(app)
 
-@app.route('/')
-def hello():
-    return "Hello World!"
+@app.route('/scrapper', methods=['GET'])
```
Comment (lilphunky) [inline src/api/app.py:26]: scraper*

Comment (areeta) [reply src/api/app.py:26]: ah ha

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/usr/local/bin/python3"
```
Comment (lilphunky) [inline .vscode/settings.json]: what does this do :o

Comment (areeta) [reply .vscode/settings.json]: im not sure where that came from but I just deleted it!

**Review by lilphunky — APPROVED** [review]

**Pull request merged**